noxfile.py

```diff
@@ -296,7 +293,7 @@ def test_plugins(session, install_cmd):
         session.chdir(os.path.join(BASE, "plugins", ray_plugin.path))
         # Running ray yield warnings
         # Make run ray tests verbose to avoid timeout
-        run_pytest(session, check_warning=False, force_verbose=True)
+        session.run("pytest", ".", silent=False)
```

Comment:
I understand why you want to do it, but I cringe seeing it.
do you know which part of the test is slow?

Reply:
Yes. So before any test is run, the test kicks off spinning up a AWS cluster, that takes around  4mins. Then after that each test takes around 20s. 

I think it probably makes sense to make the RAY tests it's standalone CI run so we can parallelize them better and save the total time to finish a whole set of CI runs. 


This is what ```$ pytest --durations=0  test_ray_aws_launcher.py```  yields

```
tests/test_ray_aws_launcher.py ....................                                                                                                                       [100%]

============================================================================ slowest test durations =============================================================================
211.35s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_1_job[pytest_dir-ray_aws-overrides0]
51.53s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_to_absolute_path_multirun[task_launcher_cfg0-extra_flags0]
26.38s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[sweep_dir_config_override-task_launcher_cfg0-extra_flags0]
26.30s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_task_name[different_filename_and_config_file_name_override_and_command_line_override-task_launcher_cfg0-extra_flags0]
26.21s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[subdir:custom_override_dirname-task_launcher_cfg0-extra_flags0]
25.84s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_task_name[different_filename_and_config_file_name_override-task_launcher_cfg0-extra_flags0]
25.79s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[sweep_dir_cli_override-task_launcher_cfg0-extra_flags0]
25.73s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[sweep_dir_cli_overridding_config-task_launcher_cfg0-extra_flags0]
25.32s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[subdir:override_dirname-task_launcher_cfg0-extra_flags0]
24.65s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_get_orig_dir_multirun[task_launcher_cfg0-extra_flags0]
24.52s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_task_name[no_config-task_launcher_cfg0-extra_flags0]
24.21s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_task_name[different_filename-task_launcher_cfg0-extra_flags0]
23.76s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_2_optimizers[pytest_dir-ray_aws-overrides0]
23.70s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_2_jobs[pytest_dir-ray_aws-overrides0]
23.42s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_over_unspecified_mandatory_default[pytest_dir-ray_aws-overrides0]
22.87s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_1_job_strict[pytest_dir-ray_aws-overrides0]
22.64s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_and_override[pytest_dir-ray_aws-overrides0]
22.38s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_not_sweeping_hydra_overrides[pytest_dir-ray_aws-overrides0]
6.45s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_task_name[no_config-task_launcher_cfg0-extra_flags0]
4.72s teardown plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_to_absolute_path_multirun[task_launcher_cfg0-extra_flags0]
1.07s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_1_job_strict_and_bad_key[pytest_dir-ray_aws-overrides0]
0.05s call     plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::test_discovery
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[sweep_dir_cli_overridding_config-task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[sweep_dir_cli_override-task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_task_name[different_filename_and_config_file_name_override_and_command_line_override-task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_2_jobs[pytest_dir-ray_aws-overrides0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[subdir:custom_override_dirname-task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_1_job_strict_and_bad_key[pytest_dir-ray_aws-overrides0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[subdir:override_dirname-task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_2_optimizers[pytest_dir-ray_aws-overrides0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_task_name[different_filename_and_config_file_name_override-task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_task_name[different_filename-task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_to_absolute_path_multirun[task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_get_orig_dir_multirun[task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayLocalLauncherIntegration::test_custom_sweeper_run_workdir[sweep_dir_config_override-task_launcher_cfg0-extra_flags0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_1_job_strict[pytest_dir-ray_aws-overrides0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_1_job[pytest_dir-ray_aws-overrides0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_over_unspecified_mandatory_default[pytest_dir-ray_aws-overrides0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_sweep_and_override[pytest_dir-ray_aws-overrides0]
0.02s setup    plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py::TestRayAWSLauncher::test_not_sweeping_hydra_overrides[pytest_dir-ray_aws-overrides0]

(0.00 durations hidden.  Use -vv to show these durations.)
======================================================================== 20 passed in 669.85s (0:11:09) =========================================================================
```